docs/wetboek.md

```diff
@@ -142,9 +142,9 @@
 7. Indien er onduidelijkheid is over medeplegen of medeplichtigheid tijdens een strafzaak dan kan een Hulpofficier van Justitie, Officier van Justitie of in uiterste gevallen een rechter hierover een eindoordeel vellen.
 
 ### A12 - Voorhechtenis
-    1. Voorhechtenis is alleen toegestaan voor feiten waar een celstraf op staat.
-    2. Voorhechtenis kan alleen gegeven worden indien er wettig overtuigend bewijs is dat de verdachten een strafbaar feit heeft begaan.
-    3. Maximale periode van de voorhechtenis is 24 uur, dit kan verlengt worden tot 48 uur door een hOvj (denk aan bijvoorbeeld aan Recherche zaken).
+1. Voorhechtenis is alleen toegestaan voor feiten waar een celstraf op staat.
+2. Voorhechtenis kan alleen gegeven worden indien er wettig overtuigend bewijs is dat de verdachte een strafbaar feit heeft begaan.
+3. De maximale periode van de voorhechtenis is 24 uur. Dit kan maximaal verlengt worden tot 72 uur voorarrest door een hOvJ.
```

Comment:
Waarom willen we voorarrest verlengen naar maximum 72u?

Reply:
@musician1011 Dat is het altijd al geweest... 24 maximaal en alleen 72 uur indien hOvJ dit goedkeurt.

Reply:
Duidelijk!